internal/report/builder.go

```diff
@@ -128,9 +128,9 @@ func (b *Builder) ToJSON() (string, error) {
 }
 
 // ToPrometheus returns prometheus pusher.
-func (b *Builder) ToPrometheus(address *string) *push.Pusher {
+func (b *Builder) ToPrometheus(address *string, cluster string, namespace string) *push.Pusher {
```

Comment:
@eminugurkenar Small pick here. dry up this method ie cluster, namespace string

Reply:
got it. doing.